### DISCUSSION TITLE
How to automatically ban bots?

### DISCUSSION THREAD

**Discussion by RobiNN1** [community/community#37793] [Other Feature Feedback, Questions, & Ideas]
Hi, is there way to automatically ban all bots like this https://github.com/deawx (spam bots too)? There is no way this is a real person, why would someone forking so much random repos.

**Comment by jfullstackdev** [comment]
maybe through the API, make a script for that 

https://stackoverflow.com/questions/52731464/can-we-use-github-api-to-determine-if-a-github-user-is-a-bot#:~:text=When%20querying%20https%3A%2F%2Fapi,type%20equals%20the%20string%20Bot%20.
